src/ComputeAngle.cpp

```diff
@@ -24,47 +24,118 @@
 
 using namespace std;
 
-/// \brief a class to compute the angle between two RSSI values.
+// enable debug messages and error messages to stderr
+#ifndef NDEBUG
+#define DEBUG
+#endif
+
+
+// a class to compute the RSSI angle for trilateration  
 class ComputeAngle
 {
+
+   private:
+   	
+   	
+   	// define what actually a, b,c stand for?
+   	unsigned int a, b, c;
+   	 	
+      // define what actually d_a, d_b, d_c stand for?
+   	float d_a, d_b, d_c;
+   	
+   	// finalise using cross calibration - done?
+      // define what actually Offset_a, offset_b,c stand for?
+   	static const int offset_a = -52;      
+   	static const int offset_b = -52;
+   	static const int offset_c = -52;
+	   
+      //define what actually N is and why 2, can we use macro or something better?
+      static const int N ;
+  
+   	/// \brief angle.
+   	float angle;
+
    public: 
-           /// \brief angle.
-           float angle;
+	
+      //constructor
+      ComputeAngle()
+      {
+            a = 0;
+            b = 0;
+            c = 0;
 
-           /// \brief a function to compute the angle.
-           // @return angle between d_a and d_b
-           float find_angle(float a, float b, float c, 
-                    float offset_a, float offset_b, float offset_c);
-};
+            d_a = 0.0;
+            d_b = 0.0;
+            d_c = 0.0;
 
-float ComputeAngle::find_angle(float a, float b, float c, 
-                     float offset_a, float offset_b, float offset_c)
-{
-   float d_a = pow(20, (a + offset_a));
-   float d_b = pow(20, (b + offset_b));
-   float d_c = pow(20, (c + offset_c));
+            N = 2;
```

Comment:
@sarthakahuja11, please look at line 53, you have declared the variable as `const static`. How can you declare a `read-only` variable with a value. In simple terms, what does `static const` means. It will give error.

Reply:
I didn't quite completely understand what you mean, but what exactly do you propose as the resolution then?